### PR TITLE
On Linux, use a AppData path inside of the home directory.

### PR DESCRIPTION
**Pull request by lonaowna**
See issue #32.

### CODE REVIEW

File: src/lib/debug.js [hunk 1]
```diff
@@ -14,7 +14,11 @@ var fileLoggerReady = false;
 // * Windows 8 - 'C:\Users\User\AppData\Roaming\Pullover'
 // * Windows XP - 'C:\Documents and Settings\User\Application Data\Pullover'
 // * Linux - '/var/local/Pullover'
```
Comment (cgrossde) [inline src/lib/debug.js]: Please adapt the comment to your change (linux path)

File: src/lib/debug.js [hunk 2]
```diff
@@ -14,7 +14,11 @@ var fileLoggerReady = false;
...
-var appDataPath = process.env.APPDATA || (process.platform == 'darwin' ? process.env.HOME + '/Library/Application\ Support' : '/var/local');
+var appDataPath = process.env.APPDATA || '/var/local';
```
Comment (cgrossde) [inline src/lib/debug.js]: I think `|| '/var/local'` is obsolete now and can be removed.

**Comment by cgrossde** [comment]
I'm currently rewriting Pullover (develop branch)  and I will incorporate this there. I don't want to release another version of Pullover before that. 
However if you need a build with your change I can provide it to you. I commented your commit, please adapt and after that I will merge this PR and send you a link to the updated binary.

**Comment by lonaowna** [comment]
I've applied your comments. I also changed the comment about the Windows path, since it uses the "new" path since Vista, and not just for 8. (only XP uses the other path)

Thanks!

**Comment by cgrossde** [comment]
I updated the linux releases here: https://github.com/cgrossde/Pullover/releases/tag/v0.3.2

Should you have some spare time then please update/correct/improve the wiki page [Installing on Linux](https://github.com/cgrossde/Pullover/wiki/Installing-on-Linux) I'm not sure if it is correct anymore.

**Comment by lonaowna** [comment]
Thanks a lot! I will take a look at the wiki page.

**Comment by lonaowna** [comment]
When trying to run the new release, I get the following error:

    [7096:1020/165552:ERROR:nw_shell.cc(336)] Error: Cannot find module 'obs'
       at Function.Module._resolveFilename (module.js:327:15)
       at Function.Module._load (module.js:264:25)
       at Module.require (module.js:356:17)
       at require (module.js:375:17)
       at window.require (<anonymous>:4:17)
       at file:///tmp/.org.chromium.Chromium.yLc2wv/js/app.js:6:11
    [7096:1020/165552:INFO:CONSOLE(329)] "Uncaught Error: Cannot find module 'obs'", source: module.js (329)

This didn't happen with the previous release... Did you change anything else? It *might* have been an system update on my side, but I don't think so.

**Comment by cgrossde** [comment]
Sorry about that, I forgot 'npm install'  after switching from the development branch -.-
I uploaded a new releases with the correct dependencies.

**Comment by lonaowna** [comment]
Works great now!
Thanks a lot!!

**Pull request merged**